packages/fx-core/src/plugins/resource/localdebug/launch.ts

```diff
@@ -524,3 +524,21 @@ export function generateSpfxCompounds(): Record<string, unknown>[] {
   );
   return launchCompounds;
 }
+
+export function getLaunchNamePrefix(launchConfig: any, isSPFxProject: boolean): string | undefined {
+  try {
+    const defaultCompoundNames = isSPFxProject
+      ? ["Teams workbench (Edge)", "Teams workbench (Chrome)"]
+      : ["Debug (Edge)", "Debug (Chrome)"];
+    const compounds = launchConfig.compounds as any[];
```

Comment:
The most accurate logic needs to check `presentation:hidden` as well...
BTW, I don't think we need to cover all corner cases here, how about just show all debug list?

Reply:
Got it, changed to show all the choices include "launch remote"